### ISSUE TITLE
AVT 1 - Carbon X Tabs Component DAP Violation

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1833]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed description
AVT 1 Run DAP on the example and there is 1 violation. See screenshot.
<img width="812" alt="screen shot 2019-02-15 at 8 18 45 am" src="https://user-images.githubusercontent.com/21676914/52862431-e5fc8680-30fa-11e9-8b1f-c466d1cd76e6.png">

Note: To fix violation add aria-disabled="true" to the 4th tab list item.

**Comment by snidersd** [comment]
Although the recommended fix will remove the violation it turns out that there is a DAP [GitHub issue 1184](https://github.ibm.com/IBMa/Tools-Rules-HTML/pull/1186) that is causing the violation in error and it should be fixed in the next DAP release expected later this month.  This should be re-tested with the new ruleset.

**Comment by snidersd** [comment]
I just noticed that the latest Tabs example has the disabled tab removed, so this issue can be closed.
